Dockerfile

```diff
@@ -1,11 +1,13 @@
-FROM jinaai/jina:2.0.0rc8
+FROM jinaai/jina:2.0.0rc6
```

Comment:
This version of base image should >= rc8, otherwise will encounter the problem of dynamic-routing error